tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.ServiceTest/Startup.cs

```diff
@@ -41,7 +41,7 @@ public Startup(IHostingEnvironment env, ILoggerFactory logger)
             this.LoggerFactory = logger;
 
             var builder = new ConfigurationBuilder()
-                .SetBasePath(@"C:\mc\mc2\tree\master\cloud\src\solution\Microsoft.Legal.MatterCenter.Service\Microsoft.Legal.MatterCenter.ServiceTest")
+                .SetBasePath(@"C:\Repos\mattercenter3\tree\master\cloud\src\solution\Microsoft.Legal.MatterCenter.Service\Microsoft.Legal.MatterCenter.ServiceTest")
```

Comment:
remove the hard coded path and use .Net api to get the path to the json file. See an example in the jobs project. Use the api such as "Directory.GetCurrentDirectory()"